packages/@ember/-internals/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.0",
   "description": "Internal APIs shared by Ember packages. Warning: this package does not follow SemVer and is subject to break at any time!",
   "type": "module",
+  "private": true,
```

Comment:
I verified that all the other package.jsons were also private.
Surprised this wasn't caught before -- but also not surprised --  because we're not using release-plan (or anything to enforce package.json consistency)

Reply:
I think for now, we could very well set up release plan for just `@glimmer/component` if we wanted.